lowcar/devices/KoalaBear/KoalaBear.cpp

```diff
@@ -189,26 +224,37 @@ size_t KoalaBear::device_write(uint8_t param, uint8_t* data_buf) {
 
         // Params for Motor B
         case DUTY_CYCLE_B:
-            this->pid_enabled_b = FALSE;  //remove later for PID functionality
             this->target_speed_b = ((float*) data_buf)[0];
```

Comment:
We briefly touched on this, but do you think it may be confusing that writing to param `duty_cycle` writes to the `target_speed` field? I can see that keeping it as `duty_cycle` is nice because that's what it's always been but with PID control (and soon acceleration shenanigans), the parameter name may not be as accurate.

Reply:
Yeah, I like this, I wanted to change it as well but wasn't sure if other people would agree with me on this. Since you also want to change the name, I'll change it!